app/assets/stylesheets/__base.scss

```diff
@@ -25,3 +25,4 @@ fieldset > h2 {
 @import "preview";
 @import "service";
 @import "tabs";
+@import "govuk";
```

Comment:
Could you move this line (28) to line 17 so that the govuk styles are loaded first, and ours overwrite them?